docs/apis-clients/operate-api/index.md

```diff
@@ -63,15 +58,13 @@ You will get something like the following:
 }
 ```
 
-Take the `access_token` value from the response object and store it as your token.
+Capture the `access_token` value from the response object. In each request to the Operate API, include it as an authorization header:
 
-2. Send the token as an authorization header in each request. In this case, request all process definitions.
-
-```shell
-curl -X POST 'http://localhost:8080/v1/process-definitions/search' -H 'Content-Type: application/json' -H 'Authorization: Bearer eyJhb...' -d '{}'
+```
+Authorization: Bearer eyJHb...
 ```
 
-#### Cookies
+#### Authentication via cookie
```

Comment:
Ifaik the cookie auth can be done only in SM environment. @ralfpuchert could you confirm please?

For C8 SaaS you can use only JWT token

Reply:
Maybe we can move it to the `Authentication for Self-Managed cluster` section and add `Use Identity JWT token to access Operate API` as a second method of authentication. @pepopowitz wdyt?

Reply:
Thanks for the info, @aleksander-dytko! 

Just to confirm I understand correctly...

What we currently have, in outline form, is:

- Authentication
   - Authentication for SaaS
      - Authentication via JWT access token
      - Authentication via cookie
   - Authentication for Self-Managed cluster (links to Operate docs)
 
What we want, in outline form, is:

- Authentication
   - Authentication for SaaS
      - Authentication via JWT access token
   - Authentication for Self-Managed cluster
      - Authentication via JWT access token
      - Authentication via cookie
 
If that's accurate, I also might think about combining the JWT access token sections, to avoid duplication. If that works out, I'd expect the outline to look something like:

- Authentication
   - Authentication via JWT access token
   - Alternatives for Self-Managed cluster
      - Authentication via cookie

Reply:
> For C8 SaaS you can use only JWT token

Yes.

Reply:
@pepopowitz yes, that's a correct structure IMO :) Can I ask you to change it directly in the docs?

Reply:
Yes, I'll follow up with a commit in this PR to adjust the structure! Thanks for the fact-checking 😄

Reply:
I added two commits to this PR, which I think should resolve this. Let me know what you think!